src/config/Config.zig

```diff
@@ -5150,23 +5150,23 @@ pub const Keybinds = struct {
         try self.set.put(
             alloc,
             .{ .key = .{ .physical = .equal }, .mods = inputpkg.ctrlOrSuper(.{}) },
-            .{ .increase_font_size = 1 },
```

Comment:
We'll have to update the macOS app to look for these to sync to the menu. Does GTK have to do the same? Or, we can just keep them the old `increase/decrease/reset` values and everything will keep working. I'd prefer that for now.